### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gaben3722**
# Pull Request Template
[Explanation Video](https://www.loom.com/share/ef693e8bbe5a441db26852ac98165763)
## Description

This PR adds the updated readme to the repo.

Fixes # (readme)

## New info added to readme

Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by imjeremiah — APPROVED** [review]
Great work following the template for PRs, including a brief explanation video. This is a simple, yet needed addition to the README, explaining the BERT model using LIME, along with supporting screenshots. No conflicts. Approved.

**Review by ahmedseragcodes — APPROVED** [review]
I think this information will be really helpful to future cohorts and it's located appropriately, approved

**Comment by iknight7000** [comment]
Adding the screenshot is incredibly helpful, and I believe it makes the end result much clearer.

**Review by ChristopherKchilton — APPROVED** [review]
The Lime explanation will be helpful for the new ML engineers later down the road. Bert is not a typical model that DS will have experience with so this is a good addition to the readme.

File: README.md [hunk 1]
```diff
@@ -59,6 +59,14 @@ This project has been worked on by many Lambda labs teams over the past 10 month
 
 The BERT model does not currently live in the GitHub repository due to its large file size. When running the app locally, it is best to manually store the `saved_model` file in the `app` directory.
 
+## BERT rankings
+Taking a deeper dive we can turn our eyes to the black box of our model. For this task we will use LIME. LIME is an acronym for local interpretable model-agnostic explanations. Local is refers to local fidelity, meaning we want the explanation to really reflect the behaviour of the classifier "around" the instance being predicted. Interpretable refers to making sense of these explanations. Lastly,  model-agnostic refers to giving explanations without needing to ‘peak’ into it.
```
Comment (paulstgermain) [inline README.md:63]: On line 32, I found a typo: 'Local is refers to local fidelity,...'

This feels as though the word 'is' was meant to be deleted. Not a merge blocker, just a nitpick for future change!

**Review by paulstgermain — APPROVED** [review]
Even though this PR is a simple change to the README, this video explanation was a little lackluster. I would have loved to hear the reason for the addition, and what problem it seeks to solve - this documentation is important for future contributors to the codebase!

That said, this is a simple change, and a well-written one at that - approved!

**Pull request merged**